sphinx/ext/napoleon/docstring.py

```diff
@@ -1340,10 +1341,7 @@ def translate(
             return []
 
         # apply type aliases
-        items = [
-            translate(func, description, role)
-            for func, description, role in items
-        ]
+        items = list(starmap(translate, items))
```

Comment:
I think Adam had a discussion about this one in a previous FURB-PR where they preferred an explicit call over `starmap`. Personally I don't mind the starmap because I'm used to parse "complex" calls and my IDE helps me a lot if I need to inspect the types, but it's maybe better to avoid using `starmap` in general.

Reply:
maybe because i'm increasingly spending my time with more functional languages, but starmap feels very natural and succinct to me

Reply:
For me it's fine, but there was a recent PR where we preferred keeping the non-starmap version. Mmh..., ok the `translate` function is close to the call and it's typed. Let's keep it.